src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/XmlJsonReader.cs

```diff
@@ -0,0 +1,1704 @@
+                CharType.None | CharType.FirstName | CharType.Name, //  D8 (O) 
+                CharType.None | CharType.FirstName | CharType.Name, //  D9 (U) 
+                CharType.None | CharType.FirstName | CharType.Name, //  DA (U) 
+                CharType.None | CharType.FirstName | CharType.Name, //  DB (U) 
+                CharType.None | CharType.FirstName | CharType.Name, //  DC (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  DD (Y) 
+                CharType.None | CharType.FirstName | CharType.Name, //  DE (_) 
+                CharType.None | CharType.FirstName | CharType.Name, //  DF (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E0 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E1 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E2 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E3 (a) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E4 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E5 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E6 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E7 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E8 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  E9 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  EA (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  EB (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  EC (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  ED (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  EE (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  EF (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F0 (d) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F1 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F2 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F3 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F4 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F5 (o) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F6 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F7 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F8 (o) 
+                CharType.None | CharType.FirstName | CharType.Name, //  F9 (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  FA (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  FB (�) 
+                CharType.None | CharType.FirstName | CharType.Name, //  FC (�) 
```

Comment:
I'm wondering what is a good way to update this. I think amending dotnet-bot remote may not get this back to Tfs so I guess I'll check in into Tfs.